### PR TITLE
zh-translation: content/zh/docs/tasks/traffic-management/ingress/gate…

### PR DESCRIPTION
**Pull request by jzhupup**
`content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md`

https://github.com/istio/istio.io/pull/11786/files

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 1]
```diff
@@ -214,3 +214,22 @@ spec:
     - name: example
       port: 80
 {{< /text >}}
+
+## 清理
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:218]:
```suggestion
## 清理 {#cleanup}
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 2]
```diff
@@ -214,3 +214,22 @@ spec:
...
+    {{< text bash >}}
+    $ kubectl delete -f @samples/httpbin/httpbin.yaml@
+    $ kubectl delete httproute http
+    $ kubectl delete gateways.gateway.networking.k8s.io gateway -n istio-ingress
+    $ istioctl uninstall -y --purge
+    $ kubectl delete ns istio-system
+    $ kubectl delete ns istio-ingress
+    {{< /text >}}
+
+1. 如果不再需要网关 API CRD 资源，请删除它们：
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:231]:
```suggestion
1. 如果不再需要这些 Gateway API CRD，请移除：
```

**Review by windsonsea — APPROVED** [review]

**Comment by windsonsea** [comment]
Please check the build log about failure reason:
```
./zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:217: MD019 Multiple spaces after hash on atx style header
```

**Pull request merged**